doc/cpp/hardware.rst

```diff
@@ -249,6 +249,18 @@ The core Arbor library *libarbor* provides an API for:
 
         The number of CPU threads available.
 
+    .. cpp:member:: bool bind_procs
+
+        Try to generate a binding mask for all processes on a node.
+        Do not set to true if process binding is handled externally,
+        eg by SLURM or OpenMPI.
+
```

Comment:
ok, but how is it useful then? I assume it would widen the mask if possible - is that what is intended?

Reply:
Nope, if SLURM made a mask, we'll live inside, thus wasting HW, so by default off and this
warning. If you want Arbor to manage this, turn off SLURMs attempt `--cpu-bind=none` and use
this. Provided for completeness.